scripts/ci/install-minikube.sh

```diff
@@ -1,98 +1,122 @@
 #!/bin/bash
 
-set -v
-
 OS=linux
 ARCH=amd64
-TARGET_DIR="/usr/bin"
+TARGET_DIR=/usr/bin
 
-MINIKUBE_VERSION="v0.24.1"
+MINIKUBE_VERSION="v0.25.2"
 MINIKUBE_URL="https://github.com/kubernetes/minikube/releases/download/$MINIKUBE_VERSION/minikube-$OS-$ARCH"
 
 KUBECTL_VERSION="v1.9.0"
 KUBECTL_URL="https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/$OS/$ARCH/kubectl"
 
-[ -z "$MINIKUBE_FLAGS" ] && MINIKUBE_FLAGS="--extra-config=kubelet.CgroupDriver=systemd"
-[ -z "$MINIKUBE_VM_DRIVER" ] && MINIKUBE_VM_DRIVER=none
+export MINIKUBE_WANTUPDATENOTIFICATION=false
+export MINIKUBE_WANTREPORTERRORPROMPT=false
+export MINIKUBE_HOME=$HOME
+export CHANGE_MINIKUBE_NONE_USER=true
+export KUBECONFIG=$HOME/.kube/config
 
 uninstall_binary() {
-	local prog=$1
-	sudo rm -f ${TARGET_DIR}/$prog
+        local prog=$1
```

Comment:
for readability of the patch would be better to keep the old indentation

Reply:
@nplanel - @lebauce asked I replace tab with space - I have separated this commits